src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -0,0 +1,506 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
```

Comment:
Are these files auto-genned? I see a lot of this stuff around that I wouldn't mind removing if we're creating these manually.